src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -133,11 +135,11 @@ def get_formatted_results(
 
         return [(serialized_groups[group_id], group_data[group_id]) for group_id in group_data]
 
-    def get(self, request: Request, group) -> Response:
+    def get(self, request: Request, group: Group) -> Response:
         if not features.has("projects:similarity-embeddings", group.project):
             return Response(status=404)
 
-        latest_event = group.get_latest_event()
+        latest_event = cast(Event, group.get_latest_event())
```

Comment:
cast should only be used for bugs in the type checker -- there's converters for `GroupEvent` from `Event` if that's what you're looking for -- or if this is an actual `GroupEvent` an `isinstance` check should be done instead